styles/variables-electro.less

```diff
@@ -64,6 +64,7 @@
 
 // Panels
 // ---------------------------------------
+@agate-panels-controls-top: 33px;
 @agate-panels-panel-padding: @agate-component-spacing;
```

Comment:
Why don't we define @agate-panels-controls-top based on @agate-panels-panel-padding like other skins to maintain our less variables easily?

```
@agate-panels-panel-padding: @agate-component-spacing;
@agate-panels-controls-top: (@agate-panels-panel-padding + 21px);
```

Reply:
I changed this in `variables-base.less`, `variables-carbon.less` and `variables-electro.less`. Please let me know if I missed other location where this needs to be changed.